### DISCUSSION TITLE
OSX AU component for Matrix 1000 panel?

### DISCUSSION THREAD

**Discussion by verticalis** [RomanKubiak/ctrlr#688] [Q&A] [answered]
Hello, 
moving to a new Mac on OSX 10.13 and I'm trying to reinstall Ctrlr functionalities, firstly a Matrix 1000 panel.
Where would I find it? I have found a 1.258 version signed by 'Poss' is it this the one the latest and the one I should use?
Many thanks to all that will reply

**Comment by damiensellier** [comment, marked as answer]
According to gearspace :  https://gearspace.com/board/electronic-music-instruments-and-electronic-music-production/1277322-oberheim-matrix-1000-ctrlr.html

This is the place you can get possemo panel:
https://roet32.wixsite.com/ctrlr/oberheim-matrix-1000

https://drive.google.com/file/d/110_L7-tqvtsFLwHsePS9Mbg4EXcP2a0t/view

Are you sure your new mac is on osx 10.13 and not macOS 13?

**Reply by damiensellier** [reply]
And a tutorial: https://ff136ad8-1f42-4ed2-8448-367ba76451fe.filesusr.com/ugd/228c20_e33c756d69cc4759b7ba99e2041f8955.pdf


https://roet32.wixsite.com/ctrlr/

And here is the link to the archived page for the Matrix 1000 panel : 
https://web.archive.org/web/20240617062935/https://ctrlr.org/matrix1000/

**Reply by verticalis** [reply]
Yep sorry, a typo, it's osx 13.6.

Thanks for links, much appreciated!

**Reply by damiensellier** [reply]
If you have any trouble let us know ;) Enjoy !
Damien

**Reply by verticalis** [reply]
thanks!!

**Reply by Possemo** [reply]
Hi, I am the one who made the Matrix1000 panel. Thanks Damien for answering the questions. Sorry I am rarely looking into ctrlr-github.

@verticalis: it would be nice if you could write here or send me an e-mail (roet32-at-gmail.com) and tell if the panel works for you. I could not test it on a real mac. I just have a virtual mac machine for developing/testing purpose.

**Reply by verticalis** [reply]
@Possemo thank you for your work! I will test it and report to you

**Reply by Possemo** [reply]
Nice, thanks in advance. I do have no idea how it performs on a Mac with Apple-Silicon CPU, in theory it should work though.

**Reply by synthmania67** [reply]
Hello Possemo, Hello Damien,

I've had a MAC MINI with an M2 processor for quite a while now. I would like to control my Oberheim Matrix 6R with the editor, but unfortunately it won't run despite all my attempts.

Dear Possemo, would it be possible to get the original panel file from your editor so that I can export it myself with my CtrlrX version?

You sent me the file by email years ago, but unfortunately I didn't archive it with the old MAC ;-(

**Reply by damiensellier** [reply]
We'll see if  Posemo replies. Otherwise I'll convert his old binaries to Ctrlrx for apple silicon when I'll have some free time.

**Reply by synthmania67** [reply]
Hi Damien, do you have the panelz or bpanelz file?

**Reply by damiensellier** [reply]
nope, I would need to extract them from the app available on Possemo website and do a little hack.

**Reply by synthmania67** [reply]
Is that possible? How?

**Reply by damiensellier** [reply]
You need to replace the executable in the available app bundle with the one you can find in Ctrlrx 5.6.33 bundle app, uncompress the panel file inside the bundle, change a string to make it unrestricted and re codesign the app or vst.

**Reply by synthmania67** [reply]
Actually, I only wanted to install the modulation matrix of the Matrix 6 for my editor. I don't need anything else from Possemos Editor, far too many bells and whistles. But that's a real challenge with this synthesizer. Anyway.

**Reply by damiensellier** [reply]
Oh ok, I'll post something this weekend. I keep it in mind.

**Reply by damiensellier** [reply]
There you go : 
[Ctrlr_Matrix1000_1_260_theArchitect_2025-05-20_16-03.bpanelz.zip](https://github.com/user-attachments/files/20349381/Ctrlr_Matrix1000_1_260_theArchitect_2025-05-20_16-03.bpanelz.zip)

**Reply by synthmania67** [reply]
Hi Damien, thank you so much.

**Reply by synthmania67** [reply]
Hi Damien, I´m sorry, but the version does not work with my Oberheim Matrix 6. There are many differences between a Matrix 1000 and the Matrix 6. Is there another version available?

**Reply by damiensellier** [reply]
https://web.archive.org/web/20240708171052/https://ctrlr.org/matrix-6/

https://roet32.wixsite.com/ctrlr/oberheim-matrix-6

_Some general information:
Keep in mind that this panel is still in beta state. Beta means that the software is not fully tested and there may be bugs which have not been discovered yet. Therefore, everybody who is using this panel is a beta-tester not an end-user. Don’t get me wrong. Bug-reports and suggestions of any kind are very welcome, but I won’t accept blaming me of having problems with your synth (e.g. losing data) as a result of using this panel. This is a risk you are taking on your own when you are using it.

Matrix-6 specific information:
The Matrix-6 panel has been developed for firmware v2.14 and above from Bob Grieb: [http://www.tauntek.com/Matrix6Firmware.htm](https://web.archive.org/web/20240614093107/http://www.tauntek.com/Matrix6Firmware.htm). It won’t work correctly when using firmware versions prior to v2.14. Notably the fixed modulations won’t work as expected. In order to get the fixed modulation working with Bob Grieb’s firmware set the “Sysex”-setting on the Matrix-6 to value “2” when using this panel. I don’t have a Matrix-6 but I’ve got many reports about the firmware and I can tell you that Bob’s firmwares (newest is v2.15) really are worth the hassle to open the Matrix’s case and replace this socketed eprom.

Important: It is not possible to send sysex-dumps to the M6’s edit buffer. This makes things a bit complicated: the panel will always OVERWRITE the actual patch location on the synth when loading a patch from the librarian and in many other cases such as using the randomizer or editing the mod-matrix. That’s why you have to save your patches if you don’t want to lose some of them. Before you try the panel better do this with a sysex tool. There are reliable freeware tools to do this. If you need help saving your patches let me know. My beta tester used patch 99 as edit buffer. The panel won’t change it when loading patches from the library so it will always stay on patch 99.

Another important thing: There is no way for the panel to know which patch nr. is set on the Matrix-6. So in order to get the same patch on the panel as on the synth you first have to use the “Patch Select” button of the panel to select a patch. Otherwise the panel will overwrite the wrong patch and you won’t hear the changes made by e.g. the randomizer or the modulation matrix.

Some time ago I made a tutorial for my SuperJX-panel. All my panels are more or less designed the same way so it may be useful to you.

The following paragraphs are somewhat applicable to all my editors. The buttons are just not at the same places as on the screenshots:

    Setting up the Ctrlr panel
    the VST-Plugin
    Saving and loading Patches (partially)_

[http://docs.wixstatic.com/ugd/228c20_f53c1f034a1c454fbd18843b1f4d39e3.pdf](https://web.archive.org/web/20240614093107/http://docs.wixstatic.com/ugd/228c20_f53c1f034a1c454fbd18843b1f4d39e3.pdf)



[Ctrlr_Matrix6_1_138_theArchitect_2025-05-21_17-11.bpanelz.zip](https://github.com/user-attachments/files/20370639/Ctrlr_Matrix6_1_138_theArchitect_2025-05-21_17-11.bpanelz.zip)

**Reply by synthmania67** [reply]
I hadn't even seen it...thanks for that. My goodness, how complicated. So many scripts...
